app/views/teams/manage.erb

```diff
@@ -132,17 +132,19 @@
       <% end %>
     </div>
 
-    <div class="row">
-      <h4>Unconfirmed members</h4>
-      <% team.unconfirmed_members.each do |member| %>
-        <div class="col-sm-3">
-          <a href="#" data-username="<%= member.username %>" data-team="<%= team.slug %>"
-          class="btn btn-xs btn-danger member_delete" style="margin: 10px 10px 10px 0">X</a>
-          <%= gravatar_tag member.avatar_url, size: 20 %>
-          <a href="/<%= member.username %>"><%= member.username %></a>
-        </div>
-      <% end %>
-    </div>
+    <% if team.member_invites.exists? %>
+      <div class="row">
+        <h4>Invited members</h4>
+        <% team.member_invites.each do |member| %>
+          <div class="col-sm-3">
+            <a href="#" data-username="<%= member.username %>" data-team="<%= team.slug %>"
```

Comment:
I've been adding inline CSS because I have no idea how to do front-end development :/ We really should get it all under control... eventually.